modules/ml/src/main/java/org/apache/ignite/ml/dataset/DatasetBuilder.java

```diff
@@ -21,8 +21,10 @@
 import org.apache.ignite.lang.IgniteBiPredicate;
 import org.apache.ignite.ml.dataset.impl.cache.CacheBasedDatasetBuilder;
 import org.apache.ignite.ml.dataset.impl.local.LocalDatasetBuilder;
+import org.apache.ignite.ml.environment.LearningEnvironmentBuilder;
 import org.apache.ignite.ml.trainers.transformers.BaggingUpstreamTransformer;
 
+
```

Comment:
There are more empty lines then we actually need.

Reply:
Fixed.